src/components/PageHeader/PageHeader.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './PageHeader.scss';
+
+const PageHeader = ( { border, children, intro, title } ) => (
+	<div className={`${styles.container} ${border ? styles.border : ''}`}>
```

Comment:
WDYT about using `<header>`? 🤔

Reply:
This sent me down a small rabbit hole. I like it generally, but MDN seems to suggest that `<nav>` should not be a child of `<header>`, which is the most obvious use case of the `children` prop for this component. I don't think that's an unforgivable sin, but would be interested to know what you think.

Reply:
Interesting, I didn't know that. Let's leave it for now, we can always revisit